sdks/java/core/build.gradle

```diff
@@ -28,12 +28,17 @@ applyJavaNature(
     dependencies {
       include(dependency("org.apache.commons:.*"))
       include(dependency(library.java.antlr_runtime))
+      include(dependency(library.java.byte_buddy))
```

Comment:
I don't think we should bundle bytebuddy in here, right? It should be a residual dependency that is resolved according to everyone's poms?

Reply:
In other words, it should become "just like any other dependency"

Reply:
Done, thanks